api/src/main/java/io/strimzi/api/kafka/model/KafkaUserQuotas.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.strimzi.crdgenerator.annotations.Minimum;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+import io.strimzi.crdgenerator.annotations.Description;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Collections.emptyMap;
+
+/**
+ * Represent the Quotas configuration for Kafka User
+ */
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode
+public class KafkaUserQuotas implements UnknownPropertyPreserving, Serializable {
```

Comment:
Can we use the `@DescriptionFile` feature to better explain the quotas or link to the Kafka docs? I think lot of people will get wrong what do these mean and how are they calculated.

Reply:
Can we do soemthing also about this? #2033 gives you an example how to use it. That would give users better explanation of how the quotas work and what do they mean.